solidity/contracts/bridge/BridgeGovernanceParameters.sol

```diff
@@ -0,0 +1,1707 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+/// @title Bridge Governance library for storing updatable parameters.
+library BridgeGovernanceParameters {
```

Comment:
How do we udpated this in the `Bridge.sol`? If there's no update function in the `Bridge.sol` for `txProofDifficultyFactor` then `BridgeGovernance` doesn't handle this scenario.

Reply:
`txProofDifficultyFactor` shouldn't be updatable.
Once set during deployment it shouldn't change the value.
We only made it settable during deployment because of tests, otherwise it could even be a constant.

Reply:
Thanks for the clarification!